app/helpers/planning_application_helper.rb

```diff
@@ -23,10 +23,6 @@ def role_name
     end
   end
 
-  def description_including_status(planning_application)
```

Comment:
realised this wasn't being used anywhere (a former PR removed the view of the line that used this) 🧹